assignment/assignment1/client/src/scenes/roomPage/editor.tsx

```diff
@@ -83,6 +85,13 @@ const Editor = ({ socket, roomId, selectedLanguage}: { socket: Socket, roomId: a
 
   return(
     <div className='editor' style={{height:"100%",overflow:"scroll"}}>
+<!--       <textarea id="realtimeEditor" placeholder="//TYPE CODE HERE"></textarea>
+      <div className='dropDownMenu'>
+        <select className="selection" value={selectedLanguage} onChange={(input) => setSelectedLanguage(input.target.value)} style={{background:"#5ee3f7", borderRadius:"10px", fontSize:"8px"}}>
+          <option value="javascript" >Javascript</option>
+          <option value="python">Python</option>
+        </select>
+      </div> -->
```

Comment:
hello can i confirm if this is outdated? if yes ill delete.

Reply:
Yes this is outdated. Can remove :)